src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyFlowGraphOperation.cs

```diff
@@ -160,7 +168,7 @@ private string GetEdgeStyle(DependencyFlowEdge edge)
         /// For more info see https://www.graphviz.org/
         /// </remarks>
         /// <returns>Async task</returns>
-        private async Task LogGraphViz(Channel targetChannel, DependencyFlowGraph graph)
+        private async Task LogGraphViz(Channel targetChannel, DependencyFlowGraph graph, bool includeBuildTimes)
```

Comment:
Can you make this LogGraphGizAsync too?

Reply:
Sure thing. You just want me to change the name of it right?

Reply:
Ya